Header header
int64 Probe_State
int64 Extended_Switch
int64 Retracted_Switch
int64 Probe_Error
